frontend/src/theme/components/DropdownMenu.ts

```diff
@@ -0,0 +1,76 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export type DropdownMenuVariant = 'outline' | 'clear'
+export type DropdownMenuSize = 'normal' | 'stretch'
+
+export const DropdownMenu: ComponentMultiStyleConfig = {
+  parts: ['outerbox', 'innerbox', 'option'],
+  baseStyle: ({ isActive }) => ({
+    outerBox: {
+      padding: '4px',
+      background: 'white',
+      borderRadius: '8px',
+      border: '0px',
+      width: '177px',
```

Comment:
need double check if fixed width or dynamic width

Reply:
thanks for highlighting - changed to dynamic width as per discussion on figma